flink-scala/src/main/scala/org/apache/flink/api/scala/DataSet.scala

```diff
@@ -89,7 +89,7 @@ class DataSet[T: ClassTag](set: JavaDataSet[T]) {
   /**
    * Returns the TypeInformation for the elements of this DataSet.
    */
-  def getType(): TypeInformation[T] = set.getType()
```

Comment:
Thanks for clarifying that it has been already discussed, my intention was to follow the suggestion of marking methods side effect free with this syntax. I am dropping the change then.